tidoop-mr-lib/README.md

```diff
@@ -1,6 +1,240 @@
-#Tidoop - MapReduce job library
+#<a name="top"></a>Tidoop - MapReduce job library
 
-##Contact
+* [What is tidoop-mr-lib](#whatis)
+* [Building](#mainbuilding)
+    * [Prerequisites](#prerequisites)
+    * [Building tidoop-mr-lib and its dependencies](#building)
+    * [Unit tests](#unittests)
+* [Installation](#install)
+* [Usage](#usage)
+    * [Running in the command line](#commandline)
+    * [Oozie-based running](#oozie)
+    * [Using tidoop-mr-lib-api](#mrlibapi)
+* [Library details](#details)
+    * [About input and output](#inputoutput)
+    * [Job categories](#categories)
+        * [Transformation and computation jobs](#transformationcomputation)
+        * [Generic and NGSI specific jobs](#genericngsi)
+        * [Flat and key-value jobs](#flatkeyvalue)
+    * [Chaining of jobs](#chaining)
+* [Jobs reference (in alphabetical order)](#jobs)
+    * [Filter](#filter)
+* [Contact](#contact)
+
+##<a name="whatis"></a>What is tidoop-mr-lib
+
+tidoop-mr-lib is a library of already developed-ready to use [MapReduce](http://research.google.com/archive/mapreduce.html) jobs for [Hadoop](https://hadoop.apache.org), allowing data scientists and engineers in genaral for an easy and quick approach to Big Data analysis.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-tidoop/commit/bb5e6d0ae4b5ea375c06376f7c17378cb6f1dd3f